components/dashboard/history-dashboard.tsx

```
@@ -403,10 +515,17 @@ function HistoryDashboardContent() {
         const slides = getPresentationSlides(item.data);
         const themeId = getPresentationThemeId(item.data);
-        return <PresentationPreview slides={slides} title={item.title} themeId={themeId} />;
+        return (
+          <PresentationPreview
+            slides={slides}
+            title={item.title}
+            themeId={themeId}
+          />
+        );
       case "generated":
         const metadata = item.data?.metadata || {};
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)